app/forms/support/recruitment_cycle_form.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+module Support
+  class RecruitmentCycleForm < ApplicationForm
+    include ActiveModel::Attributes
```

Comment:
Can we move this into `ApplicationForm`? It doesn't add complex functionality by default and we can then optionally register attributes in subclasses by calling `attribute`.